### ISSUE TITLE
Plugin breaks going to implementation of non-generic interfaces

### ISSUE THREAD

**Issue by gkinsman** [DevRust/Teapot#3]
_From @munumafia on March 31, 2015 14:29_

Assume we have the following code:

```csharp
public interface ISomeInterface
{
    // ...
}

public class SomeImplementation : ISomeInterface
{
    // ...
}
```

Right clicking on `ISomeInterface` and selecting *Go To Implementation* is a no-op. I'm running ReSharper 8.2.2 C# Edition on Visual Studio Professional 2013 

_Copied from original issue: DevRust/Teapot.Public#2_

**Comment by gkinsman** [comment]
Hi! Thanks for telling us about your issue.

We fixed this problem in the R#9 version, but haven't yet backported it to 8 due to the large changes between plugin models. If you can't upgrade to 9, then sit tight and we'll try and get a fix out ASAP. 

Thanks!

**Comment by gkinsman** [comment]
I don't think this is going to be fixed, as R#8 is now so out of date.
